README.md

```diff
@@ -108,15 +108,15 @@ With Novu, you can create custom workflows and define conditions for each channe
 
 ## 🚀 Getting Started
 
-We are excited to launch the complete Novu API and admin panel. Want to give it a test before the official release? Here is how:
+We are excited to launch the complete Novu API and admin panel. Do you want to give it a test before the official release? Here is how:
 
 ```
 npx novu init
 ```
 
 After setting up your account using the cloud or docker version, you can trigger the API using the `@novu/node` package.
 
-For API documentation and reference, please visit [Novu API Reference](https://docs.novu.co/api-reference/events/trigger-event).
+Please visit [Novu API Reference](https://docs.novu.co/api-reference/events/trigger-event) for API documentation and reference.
```

Comment:
I think we can previous version here as is it a bit more common and straightforward.